src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -4,65 +4,51 @@ import { connect } from 'react-redux';
 import { replace, Replace } from 'connected-react-router';
 import { useSpring, animated } from 'react-spring';
 import { Link } from 'react-router-dom';
+import find from 'lodash/find';
+import isEqual from 'lodash/isEqual';
+import upperFirst from 'lodash/upperFirst';
+
 import BrowserBar from './browser-bar/BrowserBar';
 import BrowserImage from './browser-image/BrowserImage';
 import BrowserNavBar from './browser-nav/BrowserNavBar';
 import TrackPanel from './track-panel/TrackPanel';
 import AppBar from 'src/shared/app-bar/AppBar';
-import find from 'lodash/find';
 
 import { RootState } from 'src/store';
-import {
-  BrowserOpenState,
-  BrowserNavStates,
-  BrowserChrLocation,
-  ChrLocation
-} from './browserState';
+import { BrowserNavStates, ChrLocation } from './browserState';
 import {
   changeBrowserLocation,
   updateBrowserNavStates,
-  updateBrowserActiveGenomeIdAndSave,
-  updateBrowserActiveEnsObjectIdAndSave
+  setDataFromUrlAndSave,
+  ParsedUrlPayload
 } from './browserActions';
 import {
-  getBrowserOpenState,
   getBrowserNavOpened,
   getChrLocation,
   getGenomeSelectorActive,
   getBrowserActivated,
   getBrowserActiveGenomeId,
   getBrowserQueryParams,
-  getBrowserActiveEnsObjectId
+  getBrowserActiveEnsObjectId,
+  getBrowserActiveEnsObjectIds,
+  getAllChrLocations
 } from './browserSelectors';
 import { getLaunchbarExpanded } from 'src/header/headerSelectors';
 import { getTrackPanelOpened } from './track-panel/trackPanelSelectors';
 import { getChrLocationFromStr, getChrLocationStr } from './browserHelper';
 import { getDrawerOpened } from './drawer/drawerSelectors';
 import { getEnabledCommittedSpecies } from 'src/content/app/species-selector/state/speciesSelectorSelectors';
 import { CommittedItem } from 'src/content/app/species-selector/types/species-search';
-import {
-  fetchEnsObject,
-  fetchEnsObjectTracks,
-  fetchExampleEnsObjects
-} from 'src/ens-object/ensObjectActions';
+import { fetchEnsObject } from 'src/ens-object/ensObjectActions';
```

Comment:
fetchEnsObject is unused.

Reply:
Will remove it for now. We'll need to think how best to fetch it if it's missing (currently it's all being done via `fetchGenomeData`, but that won't work when we start navigating to arbitrary objects from zmenu), but that can wait until the moment we actually need it.